### ISSUE TITLE
NVDA does not read the content inside if src="javascript:false" works when src="about:blank"

### ISSUE THREAD

**Issue by repl-shenoy-sukumaran** [nvaccess/nvda#11902]
### Steps to reproduce:
https://jsbin.com/tatogufiva/edit?html,output

### Actual behavior:
Its does not read the contend inside iframe if src="javascript:false" works when src="about:blank", but in third party libraries we don't have control it.

### Expected behavior:
It should read the content inside iframe

### System configuration
Windows 

#### NVDA version:
NVDA 2020.3

#### Windows version:
Windows 10

#### Name and version of other software in use when reproducing the issue:
N/A

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, not works in other version also

#### If addons are disabled, is your problem still occuring?
Yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No

**Comment by Adriani90** [comment]
@repl-shenoy-sukumaran I cannot reproduce this with NVDA 2024.2 Stable or NVDA 2024.3 Beta. Can you please describe exact steps to reproduce that you performed?
In which browsers did you test this?

**Comment by Adriani90** [comment]
We didn't get any updates from the initial author upon request since many years, I am closing as abandoned. If you are still having this issue, please comment and we can reopen.
